### PR TITLE
Download anomalies.csv file from nytimes.

### PR DESCRIPTION
**Pull request by mikelehen**
NOTE: To check if anything had changed, we used to specifically check the sha of the us-states.csv file, but now that we're ingesting anomalies.csv I'm just using the sha of the entire branch. Going back ~2 weeks I couldn't find any commits where neither us-states.csv nor anomalies.csv changed. So I think this is fine.

### CODE REVIEW

File: scripts/update_nytimes_data.py [hunk 1]
```diff
@@ -1,9 +1,6 @@
 from typing import Dict, Tuple, List
 import enum
-import logging
 import datetime
-import shutil
-import tempfile
```
Comment (mikelehen) [inline scripts/update_nytimes_data.py:6]: These were unused.

File: scripts/update_nytimes_data.py [hunk 2]
```diff
@@ -18,7 +15,6 @@
 from covidactnow.datapublic.common_fields import (
     GetByValueMixin,
     CommonFields,
-    COMMON_FIELDS_TIMESERIES_KEYS,
```
Comment (mikelehen) [inline scripts/update_nytimes_data.py:21]: Unused.

File: scripts/update_nytimes_data.py [hunk 3]
```diff
@@ -265,12 +261,14 @@ class NYTimesUpdater(pydantic.BaseModel):
...
+    ANOMALIES_CSV_DEST_FILENAME = "anomalies.csv"
     VERSION_FILENAME = "version.txt"
 
-    NYTIMES_STATES_HISTORY_API_URL = (
-        "https://api.github.com/repos/nytimes/covid-19-data/commits?path=us-states.csv"
-    )
     NYTIMES_RAW_BASE_URL = "https://raw.githubusercontent.com/nytimes/covid-19-data/master"
+    MASTER_REF_GITHUB_API_URL = (
+        "https://api.github.com/repos/nytimes/covid-19-data/git/refs/heads/master"
+    )
```
Comment (mikelehen) [inline scripts/update_nytimes_data.py:271]: We used to specifically check the sha of the us-states.csv file to see if it had changed, but now that we're ingesting anomalies.csv I'm just using the sha of the entire branch.  Going back ~2 weeks I couldn't find any commits where neither us-states.csv nor anomalies.csv changed.  So I think this is fine.

**Review by smcclure17 — APPROVED** [review]
This looks good to me! 🚢  Do you have an idea as to what the next steps are for this (how we include this in our outlier detection)?

**Comment by mikelehen** [comment]
Next step is where it gets harder. 😁   We need to have the covid-data-model snapshot build ingest these outliers and apply them to our data, either as part of or as a follow-up step to our existing drop outliers code here: https://github.com/covid-projections/covid-data-model/blob/75eeeecbb6496986af625c5aa1e9b4d59579dc90/cli/data.py#L183

I'll work on that next, though I've been taking a bit of a detour to do some cleanup work.

**Pull request merged**